client/containers/DashboardSettings/CommunitySettings.tsx

```diff
@@ -342,6 +342,24 @@ const CommunitySettings = () => {
 						/>
 					</InputField>
 				</div>
+				<div className="row-wrapper">
+					<a href="/dash/scripts">Customize CSS</a>
```

Comment:
I think we should replace this with `getDashUrl({ mode: 'scripts' })` so this doesn't break if we change the dashboard URL scheme.